### PR TITLE
Add json schema of fixture runner options

### PR DESCRIPTION
**Pull request by JLHwung**
| Q                        | A 
| ------------------------ | ---
| License                  | MIT

Add basic JSON schema of Babel fixture runners. Configure `json.schemas` so that it works on VSCode.

image

image

Note that currently `/data/schema.json` will be published to npm. I am either good with publishing or not.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/12619"></a>

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/12619"></a>

### CODE REVIEW

File: packages/babel-parser/data/schema.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
+{
```
Comment (JLHwung) [inline packages/babel-parser/data/schema.json:1]: This schema was generated from `typings/babel-parser.d.ts` by https://github.com/YousefED/typescript-json-schema. I have tuned a little bit: replacing `minItems: 2, maxItems: 2` to `additionalItems: false`.

We should explore generating API option types from JSON schema or vice.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "editor.formatOnSave": true,
```
Comment (JLHwung) [inline .vscode/settings.json:2]: This one is from my local config. I can remove it if you have different choices here. 😄

**Comment by liuxingbaoyu** [comment]
Amazing work! Looking forward to it being merged.

File: .vscode/settings.json (hunk 2 above)
Comment (nicolo-ribaudo) [reply .vscode/settings.json:2]: Yup, I have my own vscode config and the main reason I "ignored" this PR is that I was annoyed that it contained its own config, sorry :sweat_smile: 

We can rename it to `settings.example.json`, and have this in our `.gitignore`:
```
.vscode
!.vscode/settings.example.json
```

and then in `CONTRIBUTING.md` mention that we have that example vscode config.

Comment (liuxingbaoyu) [reply .vscode/settings.json:2]: https://github.com/microsoft/vscode/issues/40233

Yes, it looks like VS Code supports this a bit badly.

**Review by liuxingbaoyu — APPROVED** [review]

**Pull request merged**